CMakeLists.txt

```diff
@@ -67,27 +84,19 @@ if(FETCH_LIBHDBPP)
 
     # For ease of use, we set the libhdb++ output to our projects build directory,
     # making linking and running easy
-    set_target_properties(libhdbpp_shared_library
+    set_target_properties(${BACKEND_TARGET_SHARED}
```

Comment:
@dlacoste-esrf , It seems I managed to be in a situation where ${BACKEND_TARGET_SHARED} is empty when reaching this line, when using CMAKE with something like:
`cmake -DCMAKE_INSTALL_PREFIX=~/install/hdb++/debian9 -DCMAKE_PREFIX_PATH=/segfs/tango/release/debian9 -DFETCH_LIBHDBPP=ON -DLIBHDBPP_BACKEND=timescale  ..`

Reply:
could you try with this : -DFETCH_LIBHDBPP_TAG=project-build3

we need to merge libhdbpp-timescale as well…

Reply:
OK. Seems to work. Thanks for the info.
You can forget my comment ;-)